few/variation.py

```diff
@@ -25,7 +25,10 @@ def variation(self,parents):
                 offspring = copy.deepcopy(list(x for i,x in zip(self.ml.coef_, self.valid(parents)) if  (i != 0).any()))
```

Comment:
the parents should be weighted for this conditional as well, if self.weight_parents is True.